bpf/network_events_monitoring.h

```diff
@@ -35,10 +35,9 @@ static inline int lookup_and_update_existing_flow_network_events(flow_id *id, u8
 
     bpf_probe_read(cookie, md_len, user_cookie);
 
-    flow_metrics *aggregate_flow = bpf_map_lookup_elem(&aggregated_flows, id);
+    additional_metrics *aggregate_flow = bpf_map_lookup_elem(&additional_flow_metrics, id);
```

Comment:
nit: var name is a bit misleading here maybe flow_feature or something I assume on all other features too

Reply:
sure - in other places I called that `extra_metrics` so will use the same for consistency

Reply:
done